x-team/xp-ui
# Remove ApplicantScreenNotification animation

**Release Type:** *Non-breaking changes*

Fixes https://x-team-internal.atlassian.net/browse/XP-3442

## Description

Remove ApplicantScreenNotification animation

## Checklist


## Steps to Test or Reproduce

1. Go to the showcases:
    - http://localhost:9001/?path=/story/ui-components-applicantscreennotification--basic-usage
    - http://localhost:9001/?path=/story/screens-and-layouts-applicantscreen-states--with-notification
1. Observe that there's no longer an animation effect (see screen records below)

Additionally:

1. Copy XP-UI lib files to XP-Reg Site
1. Go to a job page and hit apply
1. Observe that the notification doesn't have animation effect and it's relatively positioned (see screen records below)

## Impacted Areas in Application

ApplicantScreenNotification

## Screenshots

![2019-11-05 08 52 46](https://user-images.githubusercontent.com/131859/68206562-c91b2280-ffab-11e9-8996-914b2d32d122.gif)

![2019-11-05 08 54 19](https://user-images.githubusercontent.com/131859/68206563-c91b2280-ffab-11e9-8df9-478e73df8024.gif)

![2019-11-05 09 05 41](https://user-images.githubusercontent.com/131859/68206564-c91b2280-ffab-11e9-9d28-e09cb09b7eb5.gif)
